openfl/component/director/experiment.py

```diff
@@ -65,10 +68,14 @@ async def start(
         """Run experiment."""
         self.status = Status.IN_PROGRESS
         try:
-            logger.info(f'New experiment {self.name} for '
-                        f'collaborators {self.collaborators}')
-
             with ExperimentWorkspace(self.name, self.archive_path):
+                if self.review_experiment:
+                    # Director to review the experiment before starting
+                    if not accept_file_post_review('plan', self.plan_path):
+                        self.status = Status.FAILED
+                        raise Exception('Experiment rejected by Director')
```

Comment:
Will this exception stop the Director service at all? We should not stop all operations because of one rejected experiment.

Reply:
it would be a reasonable move to ask Director for approval first, and only after that send the experiments to Envoys